### PR TITLE
FEM-2303: new events - dropped frames and loaded bytes

### PR DESCRIPTION
**Pull request by noamtamim**
Add two new events meant for analytics purposes:
- VideoFramesDropped(droppedVideoFrames, droppedVideoFramesPeriod, totalDroppedVideoFrames)
- BytesLoaded(bytesLoaded, totalBytesLoaded) 

Merge with https://github.com/kaltura/playkit-android-vr/pull/26.

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java [hunk 1]
```diff
@@ -423,6 +425,7 @@ public void onPlayerError(ExoPlaybackException error) {
             case ExoPlaybackException.TYPE_RENDERER:
                 errorType = PKPlayerErrorType.RENDERER_ERROR;
                 break;
+            case ExoPlaybackException.TYPE_UNEXPECTED:
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:431]: is there a point in adding this?

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java [hunk 2]
```diff
@@ -184,6 +185,7 @@ private void setPlayerListeners() {
         if (assertPlayerIsNotNull("setPlayerListeners()")) {
             player.addListener(this);
             player.addMetadataOutput(this);
+            player.addAnalyticsListener(analyticsAggregator);
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:188]: we are not removing any listeners
I think this is potential to issues.

File: playkit/src/main/java/com/kaltura/playkit/player/PlayerEngine.java [hunk 3]
```diff
@@ -239,4 +242,9 @@
     interface StateChangedListener {
         void onStateChanged(PlayerState oldState, PlayerState newState);
     }
+
+    interface AnalyticsListener {
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/PlayerEngine.java:246]: need to update VR plugin as well

**Review by giladna — APPROVED** [review]

File: playkit/src/main/java/com/kaltura/playkit/player/PlayerEngine.java (hunk 3 above)
Comment (noamtamim) [reply playkit/src/main/java/com/kaltura/playkit/player/PlayerEngine.java:246]: Thanks. https://github.com/kaltura/playkit-android-vr/pull/26.

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java (hunk 2 above)
Comment (noamtamim) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:188]: We are killing the player when the PlayerEngine is killed.

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java (hunk 1 above)
Comment (noamtamim) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:431]: Android warning.

**Pull request merged**